sci-electronics/xoscope/files/xoscope-2.2-man_no_-Tutf8.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/configure.ac b/configure.ac
+index dc79fd4..9b5220a 100644
```

Comment:
cleaned patch :wink: ?

Reply:
Done, forgot to run it through the script :smiley: